drivers/storage/nvme/NvmeIo.c

```diff
@@ -0,0 +1,99 @@
+#include "precomp.h"
+
+BOOLEAN
+NVMeIsCmdSuccessful(PNVMe_COMPLETION_QUEUE_ENTRY pCmpEntry)
+{
+    if (pCmpEntry == NULL)
+        return FALSE;
+    
+    if (pCmpEntry->DW3.SF.SC == 0 && pCmpEntry->DW3.SF.SCT == 0) {
+        return TRUE;
+    }
+
+    /* TODO: Check for retryable error we should not be failing commands */
+    ASSERT(FALSE);
+
+    DbgPrint("Failed Command CID:0x%x on SQID:0x%x SQDH:0x%x. SC: 0x%x SCT:0x%x\n", pCmpEntry->DW3.CID, 
```

Comment:
This line contains EOL-whitespaces. I will not point to all of them in each and every file you touched, but I do recommend you to use the Notepad++ feature which can fix all of them by a single click.

Reply:
Thanks for the suggestion, will do